### ISSUE TITLE
Update to API 6

### ISSUE THREAD

**Issue by crymates** [Monospark/ActionControl#5]
Please update to API 6 spongeforge

**Comment by mister-man** [comment]
Hallo,

something new?
It seems that the plugin doesnt work on mc 1.10.2 !

Thanks

**Comment by Milchshakee** [comment]
@mister-man Yeah this plugin is outdated and sadly I don't have time to update and to maintain it.
